.test_director

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -xeuo pipefail
+
+NEW_TEST=$((git diff --name-only master | grep 'tests/.*/main.yml') || true)
```

Comment:
There's been some recent work (#142 for example) to make sure our tests leave the system in the original state.  It **should** be safe to run multiple tests in succession.